crates/common/src/utils/position_utils.rs

```diff
@@ -0,0 +1,150 @@
+use crate::model::position::Position;
+use bstr::BStr;
+use bstr::ByteSlice;
+
+/// Get position of an offset in a code and return a [Position].
+pub fn get_position_in_string(content: &str, offset: usize) -> anyhow::Result<Position> {
+    if offset > content.len() {
```

Comment:
still missing `>=`

```suggestion
    if offset >= content.len() {
```

Reply:
No, if we do this, you do not catch the case where your position on the last character and will fail this [test case](https://github.com/DataDog/datadog-static-analyzer/pull/443/files#diff-5f47238d06c82545f9ce76f3c3a41dc0254731fdb43dfe210ae71156f82fe558R145)

Reply:
If you want to do this, the location will be at the end of the third line. But I think this level of detail do not really matter that much tbh